### ISSUE TITLE
Index out of bounds

### ISSUE THREAD

**Issue by magnusjjj** [micdoodle8/Galacticraft#1313]
Hi!

java.lang.ArrayIndexOutOfBoundsException: 9
        at micdoodle8.mods.galacticraft.core.tile.TileEntityElectricIngotCompressor.func_94128_d(TileEntityElectricIngotCompressor.java:419) ~[TileEntityElectricIngotCompressor.class:?]
        at appeng.util.Platform.generateTileHash(Platform.java:1520) ~[Platform.class:?]

It seems like the new code for calculating how many slots are available from which sides of the electric ingot compressor needs a tweak :). 
I reverted the code to previous commit for that function, and it works fine again.
Basically, comment out from 384 to the end of the function and replace with

return side == 0 ? new int[] { 1 } : side == 1 ? new int[] { 0, 2, 3, 4, 5, 6, 7, 8, 9, 10 } : new int[] { 0, 2, 3, 4, 5, 6, 7, 8, 9, 10 };

After 4 hours of debugging, I _think_ the error is in the calculation of the number of slots to remove, but not sure. Take a peek?
